### DISCUSSION TITLE
Tauri app / Webview

### DISCUSSION THREAD

**Discussion by mediocreatmybest** [Stability-AI/StableStudio#101] [Ideas]
Hey, 

Having a quick play of the new Tauri version of StableStudio, the stable version obviously has web access, is this going to be possible with the new Tauri version? 

As I love being able to host it on my local Linux server with ComfyUI, but this obviously limits it by only operating within a desktop app even though it's still a web app within Tauri Webview, and is now trapped in a little window. 

Cheers!

**Comment by misterteetee** [comment]
Yeah no joke - huge wish-list item that hopefully isn't omitted for non-technical reasons and/or sidelined for too long. I guess of course you could run an RDC client, but that introduces numerous hoops and hurdles (especially with image2image workflows) and whatnot. Fingers crossed...

**Reply by mediocreatmybest** [reply]
From what I've gathered, the main branch looks to be the web version that will continue forward, this will be the Tauri branch that is more a desktop app. 
The downside is the main branch doesn't have the ComfyUI back end available. 

I'm guessing they are working on this internally and then sync it to GitHub when they are ready. So it makes it a little hard to see what is being worked on or the directions etc.
